src/gtfs/read.rs

```diff
@@ -751,8 +751,12 @@ fn make_routes(gtfs_trips: &[Trip], map_line_routes: &MapLineRoutes) -> Vec<obje
         let sr = get_route_with_smallest_name(rs);
         for r in rs {
             let mut route_directions: HashSet<&DirectionType> = HashSet::new();
+            let mut geometry_id = None;
             for t in gtfs_trips.iter().filter(|t| t.route_id == r.id) {
                 route_directions.insert(&t.direction);
+                if t.shape_id.is_some() && geometry_id.is_none() {
+                    geometry_id = t.shape_id.clone();
```

Comment:
This change doesn't seems to be related to this PR.
If I understand correctly the geometry of the route will be the geometry of it's first vj that have a geom, thats right? 
if so, it's a approximation that will be false sometime, but it's a good first step.
Also I find it a bit strange to do this in the gtfs reader, I would have made that after the model to be independant of the format.